src/Controller/ContactController.php

```diff
@@ -4,26 +4,34 @@
 
 use App\Entity\Contact;
 use App\Form\ContactType;
-use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\Mime\Email;
+use Symfony\Component\Mime\Address;
+use Symfony\Component\Mailer\MailerInterface;
 use Symfony\Component\HttpFoundation\Request;
-use Symfony\Component\Routing\Annotation\Route;
 use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 
 class ContactController extends AbstractController
 {
     /**
      * @Route("/contact", name="contact")
      */
-    public function contact(Request $request): Response
+    public function contact(Request $request, MailerInterface $mailer): Response
     {
         $contact = new Contact();
         $contactForm = $this->createForm(ContactType::class, $contact);
         $contactForm->handleRequest($request);
 
         if ($contactForm->isSubmitted() && $contactForm->isValid()) {
-            return $this->render('contact/messageSent.html.twig');
+            $email = (new Email())
+            ->from(new Address('no-reply@coachmoi.com'))
+            ->to('serdar45000@gmail.com')
```

Comment:
ici il faudrait entrer l'email de la personne qui a écrit si j'ai bien compris?

Reply:
pareil pour le "to", il faut que tu le mets dans un const (pas "en dur").